src/BibleWell.App/Views/Pages/DevPageView.axaml.cs

```diff
@@ -7,5 +9,35 @@ public sealed partial class DevPageView : UserControl
     public DevPageView()
     {
         InitializeComponent();
+        
+        AttachedToVisualTree += MainView_AttachedToVisualTree;
+    }
+    private void MainView_AttachedToVisualTree(object? sender, Avalonia.VisualTreeAttachmentEventArgs e)
```

Comment:
Note that while this copies my original prototype it doesn't implement MVVM correctly.  I'll address that in a follow-up PR where this logic will move to the view model instead of the view code-behind.